src/classes/app.py

```diff
@@ -35,7 +35,7 @@
 from PyQt5.QtGui import QPalette, QColor, QFontDatabase, QFont
 from PyQt5.QtCore import Qt
 from PyQt5.QtCore import QT_VERSION_STR
-from PyQt5.Qt import PYQT_VERSION_STR
+from PyQt5.QtCore import PYQT_VERSION_STR
```

Comment:
It _shouldn't_ really matter which module this is pulled from — it's present in both, so the existing code doesn't really cause any problems. I suspect that's backwards-compatibility from all the way back in the PyQt4 days, when the documented sources for those two variables were as above. So, it's not likely to go away.

That being said, it feels _supremely_ weird (in the PyQt5 world) to be pulling `QT_VERSION_STR` from one module, but arbitrarily pulling `PYQT_VERSION_STR` from a **different** module, so purely based on THAT I'm all for this change.

In fact, on that note, my preference would be to instead replace **both** lines 37-38 with just:
```py
from PyQt5.QtCore import QT_VERSION_STR, PYQT_VERSION_STR
```

Mismatching source-module problem solved. (Whether the _previous_ line should also be subsumed into the same one-line import, I leave for better minds to discuss.)